assembly/onpremises-ide-packaging-war-factory/src/main/java/com/codenvy/onpremises/factory/deploy/FactoryServletModule.java

```diff
@@ -39,6 +41,11 @@ protected void configureServlets() {
         filterRegex(PASS_RESOURCES_REGEXP).through(com.codenvy.onpremises.factory.filter.ReferrerCheckerFilter.class);
 
 
+        Multibinder<FactoryParametersResolver> githubFactoryParametersResolverMultibinder =
```

Comment:
@garagatyi means that we never use *ServletModule to bind components. We use it only for servlet of filter mapping. It's like standard de facto. You see what I mean?